tests/tests_pytorch/callbacks/test_prediction_writer.py

```diff
@@ -111,10 +110,10 @@ def test_prediction_writer_batch_indices(num_workers):
 
 def test_prediction_writer_partial_support_for_combined_loader():
     """Test partial support for CombinedLoader: prediction works but sample indices don't get tracked."""
-    pl.loops.epoch.prediction_epoch_loop.warning_cache.clear()
 
     class PredictionModel(BoringModel):
         def predict_dataloader(self):
+            # TODO(carmocca): this should work
```

Comment:
what's this todo meaning?

Reply:
I mean that the we should support tracking the sample indices with combinedloaders. After all, it's also what we use internally during prediction after this PR.